### DISCUSSION TITLE
How to create an Uno Platform Library built on .NET 8, not .NET 7

### DISCUSSION THREAD

**Discussion by MarkBrainfire** [unoplatform/uno#14927] [General]
I tried just editing the TargetFrameworks properties from 7 to 8 and updating everything, but then the Uno templates don't produce .NET 8 compatible RIDs (eg win10-x64 => win-x64).

**Comment by limeniye** [comment]
Answered there: https://github.com/unoplatform/uno/discussions/14962#discussioncomment-8021521
